src/EnergyPlus/PackagedThermalStorageCoil.cc

```diff
@@ -2204,6 +2207,8 @@ namespace PackagedThermalStorageCoil {
 		Node( TESCoil( TESCoilNum ).EvapAirOutletNodeNum ).Temp = Node( TESCoil( TESCoilNum ).EvapAirInletNodeNum ).Temp;
 		Node( TESCoil( TESCoilNum ).EvapAirOutletNodeNum ).HumRat = Node( TESCoil( TESCoilNum ).EvapAirInletNodeNum ).HumRat;
 		Node( TESCoil( TESCoilNum ).EvapAirOutletNodeNum ).MassFlowRate = Node( TESCoil( TESCoilNum ).EvapAirInletNodeNum ).MassFlowRate;
+		Node( TESCoil( TESCoilNum ).EvapAirOutletNodeNum ).MassFlowRateMinAvail = Node( TESCoil( TESCoilNum ).EvapAirInletNodeNum ).MassFlowRateMinAvail;
+		Node( TESCoil( TESCoilNum ).EvapAirOutletNodeNum ).MassFlowRateMaxAvail = Node( TESCoil( TESCoilNum ).EvapAirInletNodeNum ).MassFlowRateMaxAvail;
```

Comment:
all other child components also pass through MassFlowRateMin(Max)Avail. Added here to conform to exiting method.